src/main/java/sixgaezzang/sidepeek/common/util/validation/TechStackValidator.java

```diff
@@ -0,0 +1,38 @@
+package sixgaezzang.sidepeek.common.util.validation;
+
+import static sixgaezzang.sidepeek.common.exception.message.CommonErrorMessage.CATEGORY_IS_NULL;
+import static sixgaezzang.sidepeek.common.exception.message.CommonErrorMessage.CATEGORY_OVER_MAX_LENGTH;
+import static sixgaezzang.sidepeek.common.exception.message.CommonErrorMessage.TECH_STACKS_OVER_MAX_COUNT;
+import static sixgaezzang.sidepeek.common.util.CommonConstant.MAX_CATEGORY_LENGTH;
+import static sixgaezzang.sidepeek.common.util.CommonConstant.MAX_TECH_STACK_COUNT;
+import static sixgaezzang.sidepeek.common.util.validation.ValidationUtils.validateMaxLength;
+import static sixgaezzang.sidepeek.common.util.validation.ValidationUtils.validateNotBlank;
+import static sixgaezzang.sidepeek.skill.exception.message.SkillErrorMessage.SKILL_ID_IS_NULL;
+
+import io.jsonwebtoken.lang.Assert;
+import java.util.List;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import sixgaezzang.sidepeek.common.dto.request.SaveTechStackRequest;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class TechStackValidator {
```

Comment:
util 클래스로 만들거면 final 키워드를 붙여도 될 것 같아용!
```suggestion
@NoArgsConstructor(access = AccessLevel.PRIVATE)
public final class TechStackValidator {
```

Reply:
오오 좋은 의견 감사해용!! 요 클래스는 상수 관리 클래스라서 상속을 하진 않을 것 같아서 final 처리하는 것이 더 깔끔하겠네용!!